releases/net471/KnownIssues/517815-BCL Applications making heavy use of System.Diagnostics.StackTrace might run more slowly on .NET 4.7.1.md

```diff
@@ -0,0 +1,19 @@
+# BCL Applications making heavy use of System.Diagnostics.StackTrace or Exception.StackTrace might run more slowly on .NET 4.7.1
```

Comment:
I think "Applications" rather than "BCL Applications" is preferable -- I'm not sure our customers think of their .NET Framework applications as BCL applications.

Reply:
I agree it sounds weird - I only wrote it that way because other issues seemed to have "BCL" or "WPF" prepended to their title.